CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+ecosystem. At current, most DEXes pull information from the _Cardano Token Registry_ but there is no similar function
+for NFT projects. As such, much information must be manually provided to individual marketplaces by the token projects
+creating an undue burden on the project creators to provide a largely static amount of information via different web
+forms and authentication schemas rather than simply publishing this information to the blockchain directly.
+
+## Specification
+
+### Registration Metadata Format
+
+A Native Script registration is a regular Cardano transaction with specific metadata attached (similar to CIP-36
+Catalyst Voter Registration). At the root level, the registration consists of two pieces of data:
+
+1. A Token Registration Payload Object [(TRPO)](#token-registration-payload-object)
+2. A Token Registration Witness array [(TRWA)](#token-registration-witness-array)
+
+### Token Registration Payload Object
+
+The Token Registration Payload Object (TRPO) consists of 4 required fields and then optional, additional fields to
+provide additional context and information as part of the registration payload. For the purposes of this draft CIP and
+in the examples below we will use the top-level JSON Metadata Index of **867** although this could be changed to the
+assigned CIP number if considered for acceptance for the sake of clarity.
+
+### Required Fields
+
+The following fields are required in all token registration submissions.
+
+#### 1. Script Hash (Policy ID)
+
+This should be the hex-encoded policy ID (**Script Hash**) that is being registered. This information is readily 
+available and accessible via Cardano DB-Sync and other chain indexers and is an easy point of reference. The script must
+include at least one key-based signature script requirement in order to witness and sign the policy registration.
+
+If a script requires multiple signatures (multi-sig) then the same validation logic should be applied when verifying
+the authenticity of the registration.
+
+**Example:**
+
+`'00000002df633853f6a47465c9496721d2d5b1291b8398016c0e87ae'`
+
+#### 2. Feature Set
+
+The Feature Set should consist of an array of CIPs that should be applied to the tokens under the given policy. For
+example, a project may specify that their tokens utilize CIP-25 (Cardano NFT Metadata Standard) and CIP-27 (Cardano NFT
+Royalty Standard) or CIP-68, or other, to be developed and determined CIPs.
+
+A registration with an empty feature set, or lack of a specific CIP, should be considered an explicit omission on behalf
+of the project and therefore any potential metadata associated with said CIPs ignored from the time of the registration
+forward.
+
+**Example:**
+
+`[25, 27]`
+
+#### 3. Witness Method
+
+The witness method portion of the payload should describe the method used to generate the witness signature (CIP-8, etc)
+to aid in validation by third parties.
```

Comment:
Might be nice to include an explanation of the flow for discovery and validation for clients.

Reply:
Agreed, this discovery flow description is currently being worked on as I am working on getting some NFTs I control on preprod so I can use the various tools involved to highlight and show various process of discovery, indexing, and validation as part of the CIP.

Reply:
I think this section needs to be expanded a bit indeed. Possibly settle on a few "well-known" methods and give concrete examples of using those authentication methods. 

One big question is how does one link that back to the original token metadata? How do we prevent someone from claiming being someone else token's owner? Do we rely on client applications to then perform manual verification of the witnesses for tokens they support?